assets/js/base/context/providers/cart-checkout/payment-methods/types.ts

```diff
@@ -66,6 +67,8 @@ export interface PaymentMethodDataContextState {
 export type PaymentMethodCurrentStatusType = {
 	// If true then the payment method state in checkout is pristine.
 	isPristine: boolean;
+	// If true then the payment method state in checkout is modified, but isn't doing anyting.
```

Comment:
I find this comment confusing, especially the ` but isn't doing anyting.` part. It's unclear to me what part is not doing anything? Does this idle state for a payment menthod happen when at the same time with this[ idle status](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/224c5f06ccf819d1d70e946af8445f5d7104a7bf/assets/js/types/type-defs/contexts.js#L110-L111)?